.github/workflows/linters/.yaml-lint.yml

```
@@ -0,0 +1,55 @@
+
+yaml-files:
```

actionlint (1.7.10)

"jobs" section is missing in workflow

(syntax-check)